tests/test_terrain.py

```
@@ -0,0 +1,122 @@
+"""
+
+import unittest
+import numpy as np
+from game.entities import Terrain
+from game import constants
+
+
+class TestTerrain(unittest.TestCase):
+    """Test the terrain generation functionality."""
+
+    def test_terrain_creation(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_terrain_creation`

Add return type annotation: `None`

(ANN201)